c_cxx/include/providers.h

```diff
@@ -2,23 +2,23 @@
 // Licensed under the MIT License.
 
 #pragma once
-#include "cpu_provider_factory.h"
+#include "cpu/cpu_provider_factory.h"
```

Comment:
In our nuget package and zip/tarball, all these headers are in the same folder. So "cpu/cpu_provider_factory.h" doesn't exist if the build directly fetch bits from our release page.

Reply:
Good point. However, it's our nuget pkg generation script that is copying the headers to a single include folder. At this point (last week of release) I'm hesitant to touch the core cmake files to install them elsewhere. The fastest option is to add the individual provider folders in the include path. At least customers won't have a bad experience running the samples. Would that work?